gui/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(REQUIRED_QT5_VERSION 5.1.0)
```

Comment:
If you're not sure, better use new version of qt. Especially, you can think about people that using fcitx 5 might use a new distro.